zerver/tests/test_home.py

```diff
@@ -459,3 +460,111 @@ def test_subdomain_homepage(self):
             with patch('zerver.views.home.get_subdomain', return_value="subdomain"):
                 result = self._get_home_page()
             self._sanity_check(result)
+
+    def soft_deactivate_user(self, user):
+        # type: (str) -> UserProfile
+        long_term_idle_user = UserProfile.objects.get(
+            email=self.example_email(user))
+        long_term_idle_user.last_active_message_id = UserMessage.objects.filter(
+            user_profile=long_term_idle_user).order_by(
+            '-message__id')[0].message_id
+        long_term_idle_user.long_term_idle = True
+        long_term_idle_user.save(update_fields=[
+            'long_term_idle',
+            'last_active_message_id'])
+        return long_term_idle_user
+
+    def get_user_ums(self, user_profile):
+        # type: (UserProfile) -> List[UserMessage]
+        return list(UserMessage.objects.filter(
+            user_profile=user_profile).order_by('message__id'))
+
+    def send_stream_message(self, content, sender_name='iago',
+                            stream_name='Denmark', subject='foo'):
+        # type: (str, str, str, str) -> None
+        sender = self.example_email(sender_name)
+        self.send_message(sender, stream_name, Recipient.STREAM,
+                          content, subject)
```

Comment:
There is no hard and fast need of this. Basically I used it to make tests look simpler and easy to read. Since doing `self.send_message()` requires passing all the same arguments repeatedly with the only difference of message I thought to just have this simple helper function in order to do just `send_stream_message(message)`. Since now sending the message is a just a one line deal tests looked less daunting and easier to read.
(Though this could be just a personal feel of things here and I would be happy to change it if that's the suggested approach)